invenio_vocabularies/records/models.py

```diff
@@ -79,7 +80,7 @@ def create(cls, **data):
         """Create a new vocabulary subtype."""
         banned = [",", ":"]
         for b in banned:
-            assert b not in data["id"], f"No '{b}' allowed in VocabularyScheme.id"
+            assert b not in data["id"], _("No '%(banned_char)s' allowed in VocabularyScheme.id", banned_char=b)
```

Comment:
minor: as i understand this message will be shown to developers in the logs only. do we translate such strings?